CacheManager.StackExchange.Redis.SelfCleaning/SelfCleaningRedisConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using CacheManager.Core;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning
+{
+    public static class SelfCleaningRedisConfigurationBuilderExtensions
+    {
+        private const string DEFAULT_CONFIGURATION_KEY = "redis";
+        private const string CONFIGURATION_KEY_SUFFIX = "-self_cleaning";
```

Comment:
When we talked you explained that this suffix has no special meaning, it exists just in order to create a different key from the one above (which has no special meaning as well), so maybe do this:

![image](https://user-images.githubusercontent.com/62211836/76998107-2ca35c80-695d-11ea-91f7-11e25acaf159.png)


I think its better beacuse the names indicates that there is no connection between those two keys.

Reply:
Changed